tests/unit/test_dt_webhook.py

```diff
@@ -0,0 +1,59 @@
+"""Unit tests for dt_webhook.py."""
+# pylint: disable-next=import-error
+import pytest
+from aiohttp import web
+
+from extensions.eda.plugins.event_source.dt_webhook import (
+    _parse_auth_header,
+    _set_app_attributes,
+)
+
+args = {
+    "host": "127.0.0.1",
+    "port": 1234,
+    "token": "thisisnotanactualtoken",
+}
+
+url = f'http://{args["host"]}:{args["port"]}/event'
+payload = {"eventId": "1A2B3C"}
+headers = {"Authorization": "Bearer " + args["token"]}
+
+
+def test_parse_token_with_incorrect_token():
+    with pytest.raises(web.HTTPUnauthorized,
+    match="Invalid authorization token"):
```

Comment:
Can this be moved into a single line instead?
```suggestion
    with pytest.raises(web.HTTPUnauthorized, match="Invalid authorization token"):
```